app/presenters/two_factor_auth_code/generic_delivery_presenter.rb

```diff
@@ -47,6 +47,14 @@ def remember_device_box_checked?
       ActiveModel::Type::Boolean.new.cast(user_opted_remember_device_cookie)
     end
 
+    def url_options
```

Comment:
Many of the presenters subclass this one, and will inherit this behavior pretty easily.